### PR TITLE
test: Add Ethereum validation tests with gas price zero

### PR DESCRIPTION
**Pull request by stoyanov-st**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/evm-functional-testing.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ permissions:
 
 env:
   SOLO_CLUSTER_NAME: solo-evm-e2e
-  MIRROR_NODE_VERSION: '0.153.0'
-  RELAY_RELEASE: '0.76.2'
+  MIRROR_NODE_VERSION: '0.158.0'
+  RELAY_RELEASE: '0.78.1'
```
Comment (gkozyryatskyy) [inline .github/workflows/evm-functional-testing.yaml:24]: mb it worth to add a comment that really pectra-upgrade is used, to avoid the confisuin

**Review by gkozyryatskyy — DISMISSED** [review]
LGTM! Really great work! Thank you @stoyanov-st !

File: .github/workflows/evm-functional-testing.yaml (hunk 1 above)
Comment (stoyanov-st) [reply .github/workflows/evm-functional-testing.yaml:24]: Added a simple comment stating this
thanks @gkozyryatskyy

**Review by gkozyryatskyy — APPROVED** [review]
LGTM! Thank you @stoyanov-st !
